code-reviews.md

```diff
@@ -0,0 +1,74 @@
+# Code reviews at FUN
+
+Code reviews are collaborative steps in software development.

```

Comment:
As it is at FUN, I wouldn't use the term **software**, if we want to add a word to describe our work, maybe I would more use project, app ? ...

```suggestion
Code reviews are collaborative steps in development.

```

Reply:
```suggestion
Code reviews are collaborative steps in software development.
Code reviews are collaborative steps in the development of our project.
```